inpage/src/EthereumRpc.ts

```diff
@@ -5,15 +5,15 @@ namespace EthereumRpc {
   export const emptyParams = z.union([z.tuple([]), z.undefined()]);
 
   export const Transaction = z.object({
-    blockHash: z.string(),
-    blockNumber: z.string(),
+    blockHash: z.union([z.string(), z.null()]),
```

Comment:
What's the purpose of `z.union([z.string(), z.null()])` in comparison to `z.string()`?

Reply:
It's the zod equivalent of `string | null` as opposed to just `string`.